.eleventy.cjs

```diff
@@ -14,6 +19,35 @@ module.exports = (eleventyConfig) => {
   });
   eleventyConfig.setUseGitIgnore(false);
 
+  const mdown = markdown({
+    html: true,
+    breaks: false,
+    typographer: true,
+  }).disable('code');
+
+  eleventyConfig.setLibrary('md', mdown);
+  eleventyConfig.addDataExtension('yaml', yaml.load);
+
+  eleventyConfig.addPairedLiquidShortcode('markdown', (content) =>
+    typogrify.typogrify(mdown.render(content)),
+  );
+
+  eleventyConfig.addPairedLiquidShortcode('markdownInline', (content) =>
+    typogrify.typogrify(mdown.renderInline(content)),
+  );
+
+  eleventyConfig.addLiquidShortcode('yearsAgo', () => {
+    return formatDistanceToNow(new Date('Tue Nov 28 19:43:58 2006 +0000'));
+  });
```

Comment:
Maybe either a more specific name that captures what date this is measuring to, or a function that can take an argument?